data/lm/generate_lm.py

```diff
@@ -191,17 +222,37 @@ def main():
     )
     parser.add_argument(
         "--discount_fallback",
-        help="To try when such message is returned by kenlm: 'Could not calculate Kneser-Ney discounts [...] rerun with --discount_fallback'",
+        help="To try when such message is returned by kenlm: 'Could not calculate Kneser-Ney "
+             "discounts [...] rerun with --discount_fallback'",
         action="store_true",
     )
 
     args = parser.parse_args()
 
+    # Basic checks on user supplied arguments
+    if args.kenlm_bins == "path/to/kenlm/build/bin/":
+        parser.error("Update kenlm_bins from documentation value to actual path for kenlm")
+
+    if not os.path.isdir(args.kenlm_bins):
+        parser.error("kenlm_bins must be a valid directory")
+
+    kenlm_files = ["lmplz","filter","build_binary"]
```

Comment:
nit: `space` after `comma` please

Reply:
Sorry, my bad! I did run it through cardboard but I missed that. Thanks!